README.md

```diff
@@ -16,7 +16,48 @@ Further documentation will be available on
 # Please note
 This version works only with `python 3.6.x`
 
+- [Get Started](#get-started)
+- [How to contribute](#how-to-contribute)
+- [Maintenance](#Maintenance)
+- [License](#license)
 
+# Get Started
 
+This is the main codebase for FARE. FARE is an e-learning web platform. Inside FARE you can found a lot of educational material, download it or you can upload some.
 
+## Searching material
+To search the desired material you just have to type what you are looking for in the search bar
+
+## Upload content
+Everyone can upload content, you have to be logged in. After the upload your content will be reviewed and, if everything is ok, it will be published
+
+## Review the material
+If you are a staff member, you can review the material and let it be visible by everyone!
+
+# How to contribute
+
+This project runs Invenio vx.x (insert structure of the project).
+To make run our instance of Invenio follow the installation [here](https://invenio.readthedocs.io/en/latest/quickstart/quickstart.html)
```

Comment:
I would state how to run *this* instance and not a Invenio instance in general

Reply:
I extended it, tell me if further improvement are needed